stacks/aws/kustomization.yaml

```diff
@@ -31,7 +31,6 @@ resources:
   # AWS components
   - ../../aws/aws-alb-ingress-controller/base_v3
   - ../../aws/nvidia-device-plugin/overlays/application
-  - ../../aws/istio-ingress/base_v3
```

Comment:
So if we deploy this in istio namespace, there's no way to put it here, right?

Reply:
Right, so this is kind-of work-around. I didn't come up with a better idea